src/components/DateRangePicker.jsx

```diff
@@ -270,10 +276,10 @@ export default class DateRangePicker extends React.Component {
   }
 
   isDayAfterHoveredStartDate(day) {
-    const { startDate, endDate } = this.props;
+    const { startDate, endDate, minimumNights } = this.props;
     const { hoverDate } = this.state;
     return !!startDate && !endDate && isNextDay(hoverDate, day) &&
-      isSameDay(hoverDate, startDate);
+      isSameDay(hoverDate, startDate) && minimumNights > 0;
```

Comment:
Yes. The next day should only appear hovered is the minimum nights is at least 1.